src/SdkCommon/ClientRuntime.Azure.Authentication/Microsoft.Rest.ClientRuntime.Azure.Authentication.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="[2.28.3]" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
-    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="[3.13.9]" />
-    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="[5.1.2]" />
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="3.17.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="5.1.4" />
```

Comment:
@FabienLavocat have you verified if this new version works with rest of the SDK.
The way SdkCommon works is, it has nuget dependency so once you build this package, can you add reference to SDKs and find out if this works.
This work has been schedule later to upgrade the entire repo not to have closed/strict dependency.
So before we roll out this, it has to be tested across.

Reply:
@shahabhijeet of course, if you have scheduled to do it later, feel free to ignore this PR.
I've built a NuGet package with this requirements and use it in my own web application. So far, I haven't noticed any problem. I'm doing queries on Resource Groups, VMs, DNS Zones.